utils/ioutil.go

```diff
@@ -56,7 +56,12 @@ func (util *impl) Download(URL string, destPath string) error {
 	}
 	defer Close(out)
 
-	resp, err := http.Get(URL)
+  client := &http.Client{}
+  req, err := http.NewRequest("GET", URL, nil)
+  req.Header.Add("X-Market-Client-Id", "VSCode 1.3.12")
+  req.Header.Add("User-Agent", "VSCode 1.3.12")
+  req.Header.Add("X-Market-User-Id" ,"007")
+  resp, err := client.Do(req)
```

Comment:
Do you plan to try whether this helps yourself or have other plans? 
FYI I tried this out a couple of week ago I have tried it a couple of weeks ago and it didn't help. See diff here https://github.com/eclipse/che-plugin-broker/commit/c8f1a5b757b6d1710260ad277ea3ff16508c1c85